src/datasources/fault-ds/mapping/ValueMapping.js

```diff
@@ -1,34 +1,10 @@
 
 export class ValueMapping {
```

Comment:
Let's keep it for now, until we know that we really don't need it. My plan is to clean up the code when we are ready for v1. I just don't want to delete it now and then re-create it later (-: